pkg/clustermanager/cluster_manager.go

```diff
@@ -329,6 +340,17 @@ func (c *ClusterManager) UpgradeCluster(ctx context.Context, managementCluster,
 		logger.V(3).Info("External etcd is ready")
 	}
 
+	logger.V(3).Info("Run provider specific upgrade operations")
+	err = c.Retrier.Retry(
+		func() error {
+			err = provider.RunSpecificUpgradeOps(ctx, currentSpec, newClusterSpec, workloadCluster)
```

Comment:
Why are we creating a new method instead of reusing `RunPostUpgrade`?

Reply:
Moving to RunPostUpgrade does not work. Moving the RunPostUpgrade function itself also does not work. Was stuck at this step.

2021-11-10T07:17:23.467-0800 V3 Reconciling CRS managed resource {"name": "vsphere-csi-controller-role"}
2021-11-10T07:17:23.467-0800 V4 Getting objects from CRS managed configmap {"name": "vsphere-csi-controller-role"}